### ISSUE TITLE
[FEATURE] League of Legend Style Emotes

### ISSUE THREAD

**Issue by Bakayarouz** [HibiscusMC/HMCCosmetics#78]
### I have checked that...

- [X] ...such a feature does not exist already
- [X] ...such a feature request has not been submitted already

### Description

So how this work would be, in term of technical
I would reckon what have to be done is summoning an armor stand that is mounted on the player head to display the custom "Emote" for an x amount of time.

https://www.youtube.com/watch?v=9eNJDztAm_U

I'm not sure if you guys want to pursue and want to make it even more indepth using ModelEngine, but I'm not sure if you can mount a model engine entity on top of the player? so we can have animated moving emotes. I guess a chicken on the head with MEG :joy:

Could be cool along with this option perhaps add a way to detect if player does certain action twice 

Shift + Right Click to display the emote or something? (Could probably apply to the current system of Emotes too)

I recall there are a wide variety of key press combo available that is detectable so that's probably something to look into for emote activation without having to do /cosmetic emote all the time

**Comment by LoJoSho** [comment]
I agree this would be a great addition to HMCC. I think implementing it into the current emote system might be best so that players can "emote" while an image is shown above them. Or they can do either one depending on how the server configures it.

**Comment by LoJoSho** [comment]
This might be a good use for displays text entities.

**Comment by Craftinators** [comment]
Rather than an emote, it could have a different name? Maybe being called an icon?

**Comment by LoJoSho** [comment]
> Rather than an emote, it could have a different name? Maybe being called an icon?

I was thinking it could be used with an emote. So a player plays an emote and it also shoots a text in the air.

**Comment by Craftinators** [comment]
Yea that would be cool! But what if you want to display something rather than text?
